src/Components/Blazor/WebAssembly.Authentication/src/AuthenticationManager.cs

```diff
@@ -28,49 +27,49 @@ public class AuthenticationManager<TAuthenticationState> : ComponentBase where T
         [Parameter] public TAuthenticationState AuthenticationState { get; set; }
 
         /// <summary>
-        /// Gets or sets a <see cref="RenderFragment"/> with the UI to display while <see cref="RemoteAuthenticationActions.Login"/> is being handled.
+        /// Gets or sets a <see cref="RenderFragment"/> with the UI to display while <see cref="RemoteAuthenticationActions.LogIn"/> is being handled.
         /// </summary>
-        [Parameter] public RenderFragment LoginFragment { get; set; } = DefaultLoginFragment;
+        [Parameter] public RenderFragment LogingIn { get; set; } = DefaultLogInFragment;
```

Comment:
Maybe this is also fixed in a later commit, but just in case it isn't:

```suggestion
        [Parameter] public RenderFragment LogingIn { get; set; } = DefaultLogInFragment;
```